src/pages/Event/AllEvents/Events.jsx

```diff
@@ -1,16 +1,16 @@
 import React, { useEffect } from "react";
-import Loading from "../../../components/Loading";
-import SingleClubEvent from "../../../components/Cards/SingleClubEvent/SingleClubEvent";
-import useSWR from "swr";
-import fetcher from "../../../utils/Fetcher";
 import Footer from "../../../components/Footer/Footer";
 import Navbar from "../../../components/Navbar/Navbar";
 import ComingSoon from "../../../components/Cards/ComingSoon/ComingSoon";
-import { eventEndpoints } from "../../../assets/data/ApiEndpoints";
 import ComponentHelmet from "../../../utils/ComponentHelmet";
+// import Loading from "../../../components/Loading";
+// import SingleClubEvent from "../../../components/Cards/SingleClubEvent/SingleClubEvent";
+// import useSWR from "swr";
+// import fetcher from "../../../utils/Fetcher";
+// import { eventEndpoints } from "../../../assets/data/ApiEndpoints";
```

Comment:
Can you please remove the code from here instead of commenting it out?

Reply:
Sure